utils/lib/connection_database.sql

```diff
@@ -252,3 +262,42 @@ CREATE TABLE constant_sources(
     FOREIGN KEY(vcc_track_pkey) REFERENCES track(pkey),
     FOREIGN KEY(gnd_track_pkey) REFERENCES track(pkey)
 );
+
+-- Grid location map.
+CREATE TABLE grid_loc_map(
+    phy_tile_pkey INT,
+    vpr_tile_pkey INT,
+    FOREIGN KEY(phy_tile_pkey) REFERENCES phy_tile(pkey),
+    FOREIGN KEY(vpr_tile_pkey) REFERENCES tile(pkey)
+);
+
+-- Tile type map.
+-- Maps physical tiles to VPR tiles. Used for CLB splitting
+CREATE TABLE tile_type_map(
+    phy_tile_type_pkey INT,
+    vpr_tile_type_pkey INT,
+    FOREIGN KEY(phy_tile_type_pkey) REFERENCES tile_type(pkey),
+    FOREIGN KEY(vpr_tile_type_pkey) REFERENCES tile_type(pkey)
+);
+
+-- Former site table
+-- Contains entries only for CLB split products. Holds information
+-- which site instance correspond to the new tile. It is necessary to know
+-- that when remapping routing resources.
+CREATE TABLE tile_former_site(
```

Comment:
Why not add a column into the the tile table? site_remap_pkey maybe.  Then site_remap_pkey == NULL means to use the phy_type connections, and site_remap_pkey != NULL means to place this site at this tile.  Clean and unambigous

Reply:
Makes sense.

Reply:
> Why not add a column into the the tile table? site_remap_pkey maybe. Then site_remap_pkey == NULL means to use the phy_type connections, and site_remap_pkey != NULL means to place this site at this tile. Clean and unambigous

I thought about this a little more, probably want this on the SLICEL tile type rather than directly on the SLICEL tile instance row.

So the remapped tile will point to the new SLICE tile type, which will have the site_remap_pkey pointing to the SLICEL site type

Reply:
I think that your first suggestion is better. Because there will be multiple VPR tiles with tile type of SLICEL that will point to different site instances.